### DISCUSSION TITLE
Showing Android Default Icon [v1.4.3]

### DISCUSSION THREAD

**Discussion by kamaravichow** [community/community#3247] [Mobile] [answered]
In version 1.4.3 of GitHub android app the icon foreground is a android default icon instead of GitHub logo.

**Comment by aconsuegra** [comment]
Thanks for the report @kamaravichow! I can't reproduce the issue that you mention so I would need a few more details from you:

- Are you using our beta build?
- What device are you using?
- What version of Android?

**Comment by aconsuegra** [comment, marked as answer]
Actually, we found the issue in our end @kamaravichow. Thanks for reporting it, we just pushed a new update fixing the app icon.

**Reply by rezababaeeeiii** [reply]
#24646
